turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -1415,18 +1420,17 @@ async fn compile_time_info_for_module_type(
     .cell())
 }
 
-async fn handle_call<G: Fn(Vec<Effect>) + Send + Sync>(
+async fn handle_call(
     ast_path: &[AstParentKind],
     span: Span,
     func: JsValue,
     this: JsValue,
     args: Vec<EffectArg>,
     state: &AnalysisState<'_>,
-    add_effects: &G,
```

Comment:
Not sure if I like that change.

It creates new Vec allocations for returning the list from handle_call, where it just pushed to the effects list before.

Can we have a `&mut Vec<Effect>` as argument where the result is pushed too instead of returning a Vec?